readme.md

```diff
@@ -37,11 +52,9 @@ Your Answer:
 Describe the differences between a SQL and NoSQL DB, and when you might use each.
 
 Your Answer:
-```text
-
 ```
-
-
+SQL is a relational database that’s divided into tables and rows and data is store in terms of schema. While NoSQL Database doesn’t have any fixed table and rows or fixed schema. It’s flexible. It poses data faster than SQL Database. No SQL database are faster because their data model are simple. If for interactive app and mobile application I will use NoSQL.
+```
```

Comment:
:+1:
(NoSQL databases can slow down dramatically when you try to impose too much pseudo-relational structure...)